src/features/practice-dashboard/hooks/usePracticeBillingData.ts

```
@@ -0,0 +1,483 @@
+      setRecentClients(Array.from(clientMap.values()).slice(0, 6));
+    } catch (err) {
+      if (signal?.aborted) return;
+      console.error('[usePracticeBillingData] Failed to load practice billing data', err);
+      setError(err instanceof Error ? err.message : 'Unable to load billing data');
+      setBillingActions([]);
+      setOutstandingSummary(null);
+      setSummaryStats([]);
+      setRecentActivity([]);
+      setRecentClients([]);
+    } finally {
+      if (signal?.aborted) return;
```

Biome (2.4.4)

[error] Unsafe usage of 'return'.

(lint/correctness/noUnsafeFinally)